src/main/java/com/xeeva/catalog/CartInformationPage.java

```diff
@@ -1,277 +1,318 @@
+		for(int rows=1; rows<=getRowsCount-1; rows++){
+			if(rows % 2 == 0){
+		       evenNum = rows;
+		       String getItemNumber = driver.findElement(By.xpath("//table[@id='tblCartInfo']/tbody/"
+		    		  + "tr["+rows+"]/td[1]/a")).getText();
+		       TestReporter.log("Updating the quantity for the item number: " + getItemNumber);
+		       driver.findElement(By.xpath("//table[@id='tblCartInfo']/tbody/"
+			       		+ "tr["+rows+"]/td[5]/input")).clear();
+		       driver.findElement(By.xpath("//table[@id='tblCartInfo']/tbody/"
+		       		+ "tr["+rows+"]/td[5]/input")).sendKeys(Quantity);
+		       btnSaveCart.syncEnabled(20);
+		       btnSaveCart.click();
+		       //Handle Alert if present
+			   if(AlertHandler.isAlertPresent(driver, 6)){
+				   AlertHandler.handleAlert(driver, 6);
+			   }
+			   verify_UpdatedQuantity();
+		       break;
+			 }
+		}
+		
+	}
+
+	/**
+	 * @summary: Method to verify the message after updating the quantity in cart-Information page.
+	 * @author: Praveen Namburi, @version: Created 28-09-2016.
+	 */
+	public void verify_UpdatedQuantity(){
+		//Added wait statement to wait till the timeout for updated quantity 
+		//successfull message to be displayed.
+		WebDriverWait wait = new WebDriverWait(driver,10);
+		WebElement lblCartAddItemMessage =wait.until(ExpectedConditions.
+				visibilityOfElementLocated(By.xpath("//div[@id='divAppInfoMsg'][@class='addMessage']")));
+		String getUpdatedQuantityMessage = lblCartAddItemMessage.getText();
+		TestReporter.logStep("Message after updating the quantity in cartInfo page: "+ getUpdatedQuantityMessage);
+		TestReporter.assertTrue(getUpdatedQuantityMessage.contains("updated successfully") || getUpdatedQuantityMessage.contains("saved successfully"), 
+				"Quantity updated successfully!");
+
+	}
+	
+	/**
+	 * @Sumamry: Method to update the Quantity and verify save cart functionality.
+	 * @author: Praveen Namburi, @Version: 28-09-2016
+	 */
+	public void deleteExistingCartItems(){
+		int evenNum = 0;  
+		pageLoaded();
+		List<WebElement> cartInfoTableRows = driver.findElements(By.xpath("//table[@id='tblCartInfo']/tbody/tr"));
+		int getRowsCount = cartInfoTableRows.size();
+		TestReporter.log("Total no. of rows in Cart Info table : " + getRowsCount);
+		//Iterate even number rows	
+		for(int rows=1; rows<=getRowsCount-1; rows++){
+			if(rows % 2 == 0){
+		       evenNum = rows;
+		       String getItemNumber = driver.findElement(By.xpath("//table[@id='tblCartInfo']/tbody/"
+		    		  + "tr["+rows+"]/td[1]")).getText();
+		       TestReporter.log("Deleting the existing Item - [" + getItemNumber + "] from cart.");
+		       driver.findElement(By.xpath("//table[@id='tblCartInfo']/tbody/tr["+rows+"]/td[8]/div/a[3]/i")).click();
+		       //Handle Alert if present
+			   if(AlertHandler.isAlertPresent(driver, 6)){
+				   AlertHandler.handleAlert(driver, 6);
+			   }
+			   verify_DeleteCartItems();
+			 }
+		   driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```

Comment:
Unless you have changed the setElementTimeout from the default timeout time (Constants.ELEMENT_TIMEOUT), there is no need to change it back to the default.  That is what it is already.  Please remove this line.